ui/src/components/SampleView/ContextMenu.jsx

```diff
@@ -538,10 +488,28 @@ export default class ContextMenu extends React.Component {
     } else {
       optionList = menuOptions.NONE.map(this.listOptions);
     }
-    return (
-      <Dropdown.Menu show id="contextMenu" role="menu">
-        {optionList}
-      </Dropdown.Menu>
```

Comment:
Note that we weren't actually using the root `Dropdown` component, which is where all the interesting features are. Of course, what matters is that we don't render a `Dropdown.Trigger` since this is not an actual "dropdown" UI element.